pkg/store/bucket.go

```diff
@@ -62,6 +62,8 @@ import (
 	"github.com/thanos-io/thanos/pkg/tracing"
 )
 
+type storeDataType string
```

Comment:
Assuming that the implementations of the limiter can be done in a separate package (eg: Cortex), shouldn't this type be exported outside this package?

Reply:
yup, will update it